speech/tests/system/gapic/v1/test_system_speech_v1.py

```diff
@@ -12,25 +12,77 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import time
+
+import os
+import io
 
 from google.cloud import speech_v1
-from google.cloud.speech_v1 import enums
-from google.cloud.speech_v1.proto import cloud_speech_pb2
 
+BUCKET = os.environ["GOOGLE_CLOUD_TESTS_SPEECH_BUCKET"]
 
 class TestSystemSpeech(object):
+
     def test_recognize(self):
 
         client = speech_v1.SpeechClient()
-        language_code = "en-US"
-        sample_rate_hertz = 44100
-        encoding = enums.RecognitionConfig.AudioEncoding.FLAC
+
         config = {
-            "language_code": language_code,
-            "sample_rate_hertz": sample_rate_hertz,
-            "encoding": encoding,
+            "encoding": speech_v1.enums.RecognitionConfig.AudioEncoding.FLAC,
+            "language_code": "en-US",
+            "sample_rate_hertz": 16000    
         }
-        uri = "gs://gapic-toolkit/hello.flac"
+
+        uri = "gs://{}/broonklyn.flac".format(BUCKET)
         audio = {"uri": uri}
+
         response = client.recognize(config, audio)
+        assert response.error.code == 200
```

Comment:
When the call fails, an exception is raised. When it is successful a `RecognizeResponse` object is returned. That object has the field`results`.  See the [code sample](https://cloud.google.com/speech-to-text/docs/quickstart-client-libraries#make_an_audio_transcription_request) for audio transcription requests.

```py
response = client.recognize(config, audio)

for result in response.results:
    print('Transcript: {}'.format(result.alternatives[0].transcript))
```

Library reference docs for the function are [here](https://googleapis.dev/python/speech/latest/gapic/v1/api.html#google.cloud.speech_v1.SpeechClient.recognize).

I don't know about the details of the speech API, but it might be nicer to make a stronger assert about `response.results`